First cut at updating for notifications (#1424)
cloudflare/cloudflare-docs: products/ssl/src/content/notifications/index.md

---
order: 200
pcx-content-type: reference
---

# Notifications

In your [account notifications](https://dash.cloudflare.com/?to=/:account/notifications), set up alerts for the following SSL areas:
- Universal SSL
- Dedicated SSL
- SSL for SaaS Custom Hostnames

These alerts cover certificate validation status, issuance, deployment, renewal, and expiration notices.

## Set up notifications

To set up notifications:
1. From your account homepage, click **Notifications**.

> 1. Select **Add**.

Reviewer: I think this is a button so should be "Click" not "Select." At some point, we agreed on using Click for CTAs and Select for choosing from options. Same goes for the last step in these instructions.

1. Select an alert type.
1. Fill in the required information and select **Create**.

> To edit, delete, or disable a notifications, you would also go to [account notifications](https://dash.cloudflare.com/?to=/:account/notifications).

Reviewer 2: ```suggestion
To edit, delete, or disable notifications, you can also go to [account notifications](https://dash.cloudflare.com/?to=/:account/notifications).
```

> To edit, delete, or disable a notifications, you would also go to [account notifications](https://dash.cloudflare.com/?to=/:account/notifications).

Reviewer: remove the "a," so "disable notifications"
Feels a bit odd to say "you would also go" in the second clause here, maybe "you can also go to" or just "go to"
